ClearKeep/Scenes/GroupCall/Views/CallingGroup.swift

```diff
@@ -0,0 +1,233 @@
+						.frame(width: Constant.paddingButtonNext, height: Constant.paddingButtonNext)
+					speakerIcon
+						.renderingMode(.template)
+						.foregroundColor(foregroundButtonSpeaker)
+				}
+			}
+			.frame(maxWidth: .infinity)
+		}
+		.frame(maxWidth: .infinity, alignment: .top)
+		.frame(minHeight: 0, maxHeight: .infinity)
+	}
+
+	var buttonCancel: some View {
+		VStack {
+			Button {
+				isTappedCancel.toggle()
+			} label: {
+				ZStack {
+					Circle()
+						.strokeBorder(backgroundEndCall, lineWidth: Constant.borderLineWidth)
+						.background(Circle().foregroundColor(backgroundEndCall))
+						.frame(width: Constant.paddingButtonNext, height: Constant.paddingButtonNext)
+					AppTheme.shared.imageSet.phoneOffIcon
+						.renderingMode(.template)
+						.foregroundColor(foregroundColorWhite)
+				}
+		}
+			Text("Call.Cancel".localized)
+				.font(AppTheme.shared.fontSet.font(style: .body2))
+				.foregroundColor(foregroundColorWhite)
+		}
+	}
+}
+
+// MARK: - Color func
+private extension CallingGroup {
+	var foregroundColorWhite: Color {
+		AppTheme.shared.colorSet.offWhite
+	}
+
+	var foregroundColorBlack: Color {
+		AppTheme.shared.colorSet.black
+	}
+
+	var foregroundColorGrey5: Color {
+		AppTheme.shared.colorSet.grey5
+	}
+
+	var foregroundColorGreyLight: Color {
+		AppTheme.shared.colorSet.greyLight
+	}
+
+	var foregroundCrossIcon: Color {
+		colorScheme == .light ? foregroundColorBlack : foregroundColorGreyLight
+	}
+
+	var backgroundGradientPrimary: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundEndCall: Color {
+		AppTheme.shared.colorSet.errorDefault
+	}
+
+	var backgroundButtonMute: Color {
+		isTappedMute == true ? foregroundColorWhite : foregroundColorWhite.opacity(0)
+	}
+
+	var foregroundButtonMute: Color {
+		isTappedMute == true ? foregroundColorBlack : foregroundColorWhite
+	}
+
+	var backgroundButtonCamera: Color {
+		isTappedCamera == true ? foregroundColorWhite : foregroundColorWhite.opacity(0)
+	}
+
+	var foregroundButtonCamera: Color {
+		isTappedCamera == true ? foregroundColorBlack : foregroundColorWhite
+	}
+
+	var backgroundButtonSpeaker: Color {
+		isTappedSpeaker == true ? foregroundColorWhite : foregroundColorWhite.opacity(0)
+	}
+
+	var foregroundButtonSpeaker: Color {
+		isTappedSpeaker == true ? foregroundColorBlack : foregroundColorWhite
+	}
+
+	var muteIcon: Image {
+		isTappedMute ? AppTheme.shared.imageSet.microphoneOffIcon : AppTheme.shared.imageSet.microPhoneIcon
+	}
+
+	var cameraIcon: Image {
+		isTappedCamera ? AppTheme.shared.imageSet.videoOffIcon : AppTheme.shared.imageSet.videoIcon
+	}
+
+	var speakerIcon: Image {
+		isTappedSpeaker ? AppTheme.shared.imageSet.speakerOffIcon : AppTheme.shared.imageSet.speakerIcon
+	}
+}
+
+struct CallingGroup_Previews: PreviewProvider {
+	static var previews: some View {
+        CallingGroup()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e7188fca22737d356920695ce8dd453fe3bcc8d0/ClearKeep/Scenes/GroupCall/Views/CallingGroup.swift#L231"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>CallingGroup.swift:231</code></td>
    </tr>
  </tbody>
</table>